### PR TITLE
readme.md with Russian translation

### PR DESCRIPTION
**Pull request by arl1nef**

### CODE REVIEW

File: README_Translations/README_ru_RU.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<p align="center">
+  <a href="https://github.com/Atlas-OS/Atlas/wiki/2.-Installing">Установка</a>
+  •
+  <a href="https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#contents">FAQ</a>
+  •
+  <a href="https://discord.com/servers/atlas-795710270000332800" target="_blank">Discord</a>
+</p>
+<p align="center">
+ Другие языки:
+  <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_zh_CN.md">简体中文</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_fr_FR.md">Français</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_pl_PL.md">Polski</a>
```
Comment (JayXTQ) [inline README_Translations/README_ru_RU.md:16]:
```suggestion
  <a href="https://github.com/Atlas-OS/Atlas/blob/main/README.md">English</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_zh_CN.md">简体中文</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_fr_FR.md">Français</a> • <a href="https://github.com/Atlas-OS/Atlas/blob/main/README_Translations/README_pl_PL.md">Polski</a>
```

**Review by JayXTQ — APPROVED** [review]
looks fine to me, [DeepL translate](https://www.deepl.com/translator) didn't translate all of it on my end but yeah

File: README_Translations/README_ru_RU.md (hunk 1 above)
Comment (JayXTQ) [reply README_Translations/README_ru_RU.md:16]: Oh yeah and also, consider changing this, linking back to English readme

**Comment by SyrmGIT** [comment]
Need to add links to Russia in all of the readme's which exist.

**Review by Xyueta — APPROVED** [review]
manually checked, all good

**Pull request merged**